node-backend/app/controllers/v1/recognitionController.js

```diff
@@ -0,0 +1,240 @@
+const moment = require("moment"); //eslint-disable-line node/no-extraneous-require
```

Comment:
I don't see this package in package.json

Reply:
it is present in package.json dependencies below of jwt-decode package

Reply:
ok